packages/dns-discovery/karma.conf.cjs

```diff
@@ -1,5 +1,9 @@
-process.env.CHROME_BIN = require("puppeteer").executablePath();
 const webpack = require("webpack");
+const playwright = require('playwright');
+
+process.env.CHROME_BIN = playwright.chromium.executablePath();
```

Comment:
I am not sure why I kept the `karma.conf.cjs` file dupe across all packages.

There might be some slight difference but surely it should not be too hard to have most of the config in a common file imported here, no?

Reply:
Same here, tried to move it up and something got broken so proceeded with it, but yeah, should be similar to typescript.